### DISCUSSION TITLE
Integrate Selfh.st Icons

### DISCUSSION THREAD

**Discussion by hxck** [Sanjeet990/Astroluma#7] [Ideas]
Integrate icons from the [Selfhst](https://selfh.st/icons) project in a similar way to some other projects, so the user can just add sh-docmost etc in the icon input and automatically retrieve service icons. Would go a long way in getting user adoption.

**Comment by Sanjeet990** [comment]
This is surely awesome man. I myself as a user would love that. Thanks for the suggestion, I'll put that in list.

**Reply by Sanjeet990** [reply]
Implemented in https://github.com/Sanjeet990/Astroluma/pull/25/commits/4c3c161d367bc0ad81d21a52115a44d536640045

Not just Selfh.st, Astroluma will support multiple icon packs now. Icon packs are coming soon on portal.
